Running tests under Python 3.9.18: /home/jmw/anaconda3/envs/vct_tf/bin/python
[ RUN      ] ModelsTest.test_session
[  SKIPPED ] ModelsTest.test_session
[ RUN      ] ModelsTest.test_train_eval
INFO:tensorflow:time(__main__.ModelsTest.test_train_eval): 0.83s
I1121 15:59:29.880537 139693896540544 test_util.py:2460] time(__main__.ModelsTest.test_train_eval): 0.83s
[  FAILED  ] ModelsTest.test_train_eval
======================================================================
ERROR: test_train_eval (__main__.ModelsTest)
ModelsTest.test_train_eval
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jmw/ing/myfi/vfi/src/models_test.py", line 74, in test_train_eval
    model= Model(feature_extractor(**feature_extractor_cfg), **flow_estimation_cfg)
  File "/home/jmw/ing/myfi/vfi/src/model.py", line 189, in __init__
    self.optimizer = tfa_optimizers.AdamW(learning_rate= self.learning_rate_schedule)
TypeError: __init__() missing 1 required positional argument: 'weight_decay'
----------------------------------------------------------------------
Ran 2 tests in 0.828s
FAILED (errors=1, skipped=1)